source/operator.rst

```diff
@@ -171,546 +317,707 @@
 | **Value**       | string                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``dynamic``                                                                               |
+| **Example**     | ``""``                                                                                    |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Type of the PostgreSQL write-ahead log storage: ``create`` (by default) or ``dynamic``    |
+| **Description** | `Labels are key-value pairs attached to objects                                           |
+|                 | <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_             |
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
-|                 | .. _walstorage-supplementalgroups:                                                        |
+|                 | .. _pgreplicas-labels:                                                                    |
 |                 |                                                                                           |
-| **Key**         | `WALStorage.supplementalgroups <operator.html#walstorage-supplementalgroups>`_            |
+| **Key**         | `pgReplicas.<replica-name>.labels <operator.html#pgbouncer-labels>`_                      |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Value**       | string                                                                                    |
+| **Value**       | label                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``""``                                                                                    |
+| **Example**     | ``pg-cluster-label: cluster1``                                                            |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | Supplemental groups for the PostgreSQL write-ahead log storage                            |
+| **Description** | `Labels are key-value pairs attached to objects                                           |
+|                 | <https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/>`_             |
```

Comment:
Again the same template, but the description would be different here. This one actually sets the labels for pgReplica Pods.